mypyc/irbuild/env_class.py

```diff
@@ -191,6 +191,41 @@ def add_args_to_env(
                 builder.add_var_to_env_class(arg.variable, rtype, base, reassign=reassign)
 
 
+def add_vars_to_env(builder: IRBuilder) -> None:
```

Comment:
Move this code here since this is used for both regular and generator functions.